client/examples/quickstart.py

```diff
@@ -0,0 +1,77 @@
+# Run in command line with:
+# python3 -m featureform examples/cli.py -host <hostname:port>
```

Comment:
should be python3 -m featureform apply examples/quickstart.py --host <hostname:port>